src/Banner.js

```diff
@@ -5,8 +5,9 @@ const Banner = () => {
     return (
       <div className='Banner'>
         <div className='banner-center'>
-          <img src='./images/icecreams.jpg' className='cones-img'/>
-          <p className='banner-text'>Our mission is to show you the craziest, most interesting, new ice cream flavors on the block around Denver and Aurora. You can use the search bar to locate your favorite ice cream or just browse our homepage to see the coolest new flavors!</p>
+          <img src="http://purepng.com/public/uploads/large/purepng.com-ice-cream-conefood-ice-cream-ice-cream-cone-cone-scoop-vanilla-ice-941524636426f7rep.png"
```

Comment:
I think this image looks good with the transparent background and I like the different ice cream colors. We can decide as a group tomorrow which img we would like to use. Thanks for finding this one